### ISSUE TITLE
JPEG image.MetaData.Properties Collection Empty

### ISSUE THREAD

**Issue by ian-a-anderson** [SixLabors/ImageSharp#850]
### Prerequisites

- [x] I have written a descriptive issue title
- [x] I have verified that I am running the latest version of ImageSharp
- [x] I have verified if the problem exist in both `DEBUG` and `RELEASE` mode
- [x] I have searched [open](https://github.com/SixLabors/ImageSharp/issues) and [closed](https://github.com/SixLabors/ImageSharp/issues?q=is%3Aissue+is%3Aclosed) issues to ensure it has not already been reported

### Description
When loading JPEG images with a "Title" property (exported from macOS Photos app) the image.MetaData.Properties collection is empty. I expected at least one property to be loaded, with the Name "Title" and the Value matching the value set in the JPEG.

### Steps to Reproduce
Load the attached file, which has a title set to "Kitchen Appliances", and note the empty image.MetaData.Properties collection. Compare to Windows File > Properties > Details info that shows the Title as expected, and macOS File > Get Info that also shows the Title (see attached screenshots).

NUnit repo code: 
```
using (var image = Image.Load("C:\\IMG_1280.jpg")) {
     TestContext.Out.WriteLine($"image.MetaData.Properties = {image.MetaData.Properties.Count}"); // expected, > 0
     foreach (var prop in image.MetaData.Properties) {
          TestContext.Out.WriteLine($"prop = {prop.Name} = {prop.Value}"); // expected Title property
     }
}
```

### System Configuration

- ImageSharp version: 1.0.0-beta0006
- Other ImageSharp packages and versions:  Also occurs in beta0005 and beta0004 (didn't test older versions)
- Environment (Operating system, version and so on): Windows 10 Pro v1809
- .NET Framework version: .Net Core 2.2
- Additional information:

### Attachments
IMG_1280:
![IMG_1280](https://user-images.githubusercontent.com/2657265/54477199-eb252200-47db-11e9-8740-90a355313ef5.jpg)

macOS File > Get Info (showing Title):
<img width="400" alt="macOS-Get-Info" src="https://user-images.githubusercontent.com/2657265/54477217-23c4fb80-47dc-11e9-9f1e-87ca6635195a.png">

Windows 10 File > Properties > Details (showing Title):
<img width="361" alt="Windows10-Details" src="https://user-images.githubusercontent.com/2657265/54477229-4bb45f00-47dc-11e9-8580-b951f7464da3.png">

**Comment by JimBobSquarePants** [comment]
According to jpegsnoop this data is stored in th App13 tag which we don't decode just now.

![image](https://user-images.githubusercontent.com/385879/54718548-fd72c900-4b17-11e9-9c1a-3542ca8efa61.png)

**Comment by ian-a-anderson** [comment]
I ultimately was able to get at this info via [MetadataExtractor](https://github.com/drewnoakes/metadata-extractor-dotnet). Since this data appears in the IPTC header maybe this request should be merged into #860? Thanks!

**Comment by JimBobSquarePants** [comment]
Agreed. Let's pick it up in #860
